.travis.yml

```diff
@@ -0,0 +1,6 @@
+osx_image: xcode7.3
+language: node_js
+node_js:
+  - "4.4.x"
+script:
+  - npm install
```

Comment:
I'm not sure if Linux builds should be generated on Travis too to have them integrated... What do you think?